src/pages/solutions/for-students.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: We help students 
+quickLinks:
+  - name: Data
+    anchor: data
+  - name: Basemaps and Imagery
+    anchor: imagery
+  - name: Conferences and Events
+    anchor: conferences
+  - name: Jobs and Internships
+    anchor: jobs
+  - name: Metadata
+    anchor: metadata
+  - name: Citations
+    anchor: citations
+
+---
+
+import Layout from '../../layouts/FullWidth.astro';
+import Section from '../../components/page/Section.astro';
+import QuickLinks from '../../components/page/QuickLinks.astro';
+
+<Layout title={frontmatter.title}>
+  <QuickLinks links={frontmatter.quickLinks}/>
+  <Section stripe title="Data">
+
+    `Data! Data! Data! I can't make bricks without clay!` - Sherlock Holmes
```

Comment:
ticks are for code > is for quotes

```suggestion
    > Data! Data! Data! I can't make bricks without clay! - Sherlock Holmes
```

Reply:
The quote isn't formatting any different than the rest of the page. Any ideas on why?

Reply:
There are no default styles for markdown output. In this case blockquote.

Reply:
i added a component that we can iterate on